app/houston/parameters.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+"""
+RESTful API Parameters base class with password protection
+--------------------------
+"""
+from flask_login import current_user
+from flask_restplus_patched import PatchJSONParameters
+from flask_restplus._http import HTTPStatus
+
+from app.extensions.api import abort
+
+
+# noinspection PyAbstractClass
+class PatchJSONParametersWithPassword(PatchJSONParameters):
```

Comment:
Let's consolidate the location of `PatchJSONParametersWithPassword` and `PatchJSONParameters` by putting them both in `flask_restplus_patched/parameters.py`.  This adds an import from outside of `flask_restplus_patched` into `app.extensions.api` for `abort` but that is acceptable in this case